client/my-sites/stats/activity-log-day/index.jsx

```diff
@@ -97,15 +104,20 @@ class ActivityLogDay extends Component {
 	getEventsHeading() {
 		const {
 			applySiteOffset,
+			isToday,
 			logs,
 			moment,
 			translate,
 			tsEndOfSiteDay,
 		} = this.props;
 
+		const formattedDate = applySiteOffset( moment.utc( tsEndOfSiteDay ) ).format( 'LL' );
+
 		return (
 			<div>
-				<div className="activity-log-day__day">{ applySiteOffset( moment.utc( tsEndOfSiteDay ) ).format( 'LL' ) }</div>
+				<div className="activity-log-day__day">
+					{ isToday ? translate( '%s — Today', { args: formattedDate } ) : formattedDate }
```

Comment:
Could be useful for translators to describe in a comment what the argument contains.